src/pages/tinker/tinker.jsx

```diff
@@ -172,12 +173,19 @@ var Tinker = React.createClass({
   },
   setEditorValue: function (value) {
     var edits = {};
+    this.preventInvisibleColors(value);
     edits[this.state.params.propertyName] = value.rgbaString();
     this.setState({
       element: assign({}, this.state.element, edits)
     });
   },
-
+  preventInvisibleColors: function(value) {
+    var currentColor = Color(this.state.element.color).values;
+    var nextColor = value.values;
+    if(currentColor.alpha == 0 && nextColor.alpha == 0){
```

Comment:
Though since it's coming from the Color module, that's highly unlikely. I'll change it.